Can't Install w/o husky

There seems to be a recent change that is causing installing my OCLIF CLI to break if husky is not installed on the system.

When running

```sh
npm install -g mycli
```

I'm seeing the following error

```sh
| #6 10.55 npm ERR! code 127
| #6 10.55 npm ERR! path /usr/local/lib/node_modules/griffin-cli/node_modules/@oclif/plugin-version
| #6 10.55 npm ERR! command failed
| #6 10.55 npm ERR! command sh -c husky install
| #6 10.55 npm ERR! sh: 1: husky: not found
| #6 10.55 
```

It looks like it could be related to [this change](https://github.com/oclif/plugin-version/commit/2a4ac65593e9e9bbf20d2ccc4d6e35da88be9bea).

If I do install `husky`, I get a different error

```sh
npm ERR! code 1
npm ERR! path /usr/local/lib/node_modules/griffin-cli/node_modules/@oclif/plugin-version
npm ERR! command failed
npm ERR! command sh -c husky install
npm ERR! husky - .git can't be found (see https://typicode.github.io/husky/#/?id=custom-directory)
```

--- a/package.json
+++ b/package.json
@@ -59,7 +59,7 @@
     "lint": "eslint . --ext .ts",
     "postpublish": "yarn run clean",
     "posttest": "yarn lint",
-    "postinstall": "husky install",
+    "prepack": "husky install",
     "prepublishOnly": "yarn run build && oclif lock && oclif manifest . && oclif readme",
     "pretest": "yarn build --noEmit && tsc -p test --noEmit",
     "preversion": "yarn run clean",

